src/style.css

```diff
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial ;
+  font-size: 13px;
+}
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 1em;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+}
+
+a {
+  color: green;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
Міша сказав робити стилі через теги.

Reply:
don't mind about tests, make it using classes

Reply:
ок

Reply:
Я переробив через класи. Глянь будь-ласка.дякую